src/Docker.DotNet/Models/ContainerExecInspectResponse.Generated.cs

```diff
@@ -5,7 +5,7 @@ namespace Docker.DotNet.Models
     [DataContract]
     public class ContainerExecInspectResponse // (types.ContainerExecInspect)
     {
-        [DataMember(Name = "ExecID", EmitDefaultValue = false)]
+        [DataMember(Name = "ID", EmitDefaultValue = false)]
```

Comment:
Out of curiosity, why are we using `DataMember` instead of `JsonProperty` on this case?

Reply:
It's output like that by specgen

https://github.com/dotnet/Docker.DotNet/blob/a5ee0611f37711ec81fbdb5dc48f83e642da2a2a/tools/specgen/specgen.go#L601#L607

Reply:
Ahhh yeah. I forgot it still Go-based. Sorry for the confusion, my branch doesn't use it anymore. Ignore what I said :) thanks!